editor/plugins/animation_state_machine_editor.cpp

```diff
@@ -1946,6 +1946,8 @@ AnimationNodeStateMachineEditor::AnimationNodeStateMachineEditor() {
 
 	delete_window = memnew(ConfirmationDialog);
 	delete_window->set_flag(Window::FLAG_RESIZE_DISABLED, true);
+	delete_window->set_flag(Window::FLAG_MINIMIZE_DISABLED, true);
+	delete_window->set_flag(Window::FLAG_MAXIMIZE_DISABLED, true);
```

Comment:
ConfirmationDialog is an AcceptDialog, so these are already true by default.
Also I think the delete_window is never used. #75759 removed the only time it is shown.